tests/benchmarks/multicore/idle_pwm_loopback/remote/Kconfig

```
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config TEST_SLEEP_DURATION_MS
+	int "Amount of time (in miliseconds) the core is sleeping"
```

Minor nit, not blocking.

Perhaps shorten this to:
```suggestion
	int "Core sleep duration (miliseconds)"
```
after all this is a title and details are in the help text.

---

Fixed